### PR TITLE
[TASK] Remove old wiki links

### PR DESCRIPTION
**Pull request by linawolf**
refs https://github.com/TYPO3-Documentation/T3DocTeam/issues/170

releases: main, 11.5, 10.4, 9.5, 8.7, 7.6

### CODE REVIEW

File: Documentation/Major/PostupgradeTasks/BackendTranslations.rst.txt [hunk 1]
```diff
@@ -5,9 +5,8 @@
 In the Install tool, go to the module "Maintenance" -> "Manage languages" and
 update your translations. If you don't update your translations, new texts will
 only be displayed in English. The translations are available, once the
-`Translation Team`_ for your language has translated the texts.
+:ref:`Translation Team <t3coreapi:internationalization>` for your language has
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Major/PostupgradeTasks/BackendTranslations.rst.txt:8]: There is the Localization with Crowdin Initiative https://typo3.org/community/teams/typo3-development/initiatives/localization-with-crowdin/ should we link to that? But not sure if the workflow changed

[the remark this answers is not in the document]
Comment (alexander-nitsche) [reply Documentation/Major/PostupgradeTasks/BackendTranslations.rst.txt]: Subpage [Localization with Crowdin](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Internationalization/TranslationServer/Crowdin.html) still talks a lot of the translation team and links to the Slack channel _#cig-crowdin-localization_. Would leave the reference URL as is, but remove the team from the reference name. Added a proposal in the following commit.

[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Major/PostupgradeTasks/BackendTranslations.rst.txt]: I am talking to the team directly now, we probably will have to reformulate the section all together

**Comment by linawolf** [comment]
I think the change is fine for now, I will make another PR for additional necessary changes. Could you approve @alexander-nitsche  as I created the PR I cannot approve it.

**Review by alexander-nitsche — APPROVED** [review]

**Pull request merged**